iam-login-service/src/main/java/it/infn/mw/iam/api/scim/new_updater/UpdaterType.java

```diff
@@ -1,23 +0,0 @@
-package it.infn.mw.iam.api.scim.new_updater;
```

Comment:
I don't like commits that destroy history and authorship. It seems git here didn't understand that the file was moved from one directory to another. Let's try to avoid this kind of errors in the future